tests/ops/math/test_flatten.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the operator math utils.
+"""
+import pennylane as qml
+from pennylane.ops.math.utils import flatten_decomposition, flatten_terms
+from numpy import np
+
+
+def test_flatten_decomposition():
```

Comment:
Used these to develop the methods and thought I'd add them here for future reference